src/pages/auth/Login.jsx

```diff
@@ -0,0 +1,126 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuthContext } from "../../context/useContext-index";
+
+export const Login = () => {
+  const navigate = useNavigate();
+  const { authDispatch } = useAuthContext();
+  const [loginData, setLoginData] = useState({
+    email: "",
```

Comment:
suggestion: send the useState to auth-context

Reply:
the login form will not be used anywhere else, so I don't think it requires passing in context!